### ISSUE TITLE
Windows should retain their relative sizes in Scale mode

### ISSUE THREAD

**Issue by Hubro** [linuxmint/cinnamon#3336]
It would be a lot easier to quickly find the right window in Scale mode if the window had retained it's size relative to the other windows. Currently, a maximized window will have the same size in Scale mode as a tiny dialog box.

If you have 2 or more windows open in the same application, say Chrome, it can be very time consuming to pick out the correct window every time you use Scale. This would be a lot easier if there was an option to (at least partly) retain the window sizes in Scale mode.

This is (approximately) how Scale works now:

![scale-old](https://cloud.githubusercontent.com/assets/597206/3489520/39fcc568-0533-11e4-8b5f-8e44e708858f.png)

I suggest it should work (perhaps optionally?) like this:

![scale-suggestion](https://cloud.githubusercontent.com/assets/597206/3489523/5c52075e-0533-11e4-9b2c-d65116480924.png)

This makes it much easier to quickly pick our the maximized browser instance from all the smaller ones, as well as finding the small message boxes that popped up in the background, etc.

**Comment by tp0** [comment]
+1. This causes a visual distraction.

If I remember correctly, good (bad) old Compiz had a some kind of choice between "natural" and some other option in its Scale plugin which also caused the distance between the windows to be too great, but the idea was right.

**Comment by quicksi1vr** [comment]
Agree. They should also hold their position until they are closed / new windows are opened, instead of changing the order based on the currently picked window [#3906](https://github.com/linuxmint/Cinnamon/issues/3906).
